crypto/ec/curve448/arch_32/f_impl.h

```diff
@@ -10,8 +10,8 @@
  * Originally written by Mike Hamburg
  */
 
-#ifndef HEADER_ARCH_32_F_IMPL_H
-# define HEADER_ARCH_32_F_IMPL_H
+#ifndef OSSL_CRYPTO_F_IMPL_H
+# define OSSL_CRYPTO_F_IMPL_H
```

Comment:
Sometimes, it might be worth considering hand-crafting.  I know the joy of writing the script that automates everything, but you have to consider the time spent and how much value that gives.

Ah well, it's your time after all ;-9

Reply:
Apart from the joy of writing the, the 100% automation ratio gives me the advantage of being able to rebase without effort on  master as well as on the 1.1.1 stable branch. But of course for the fine tuning at the end it will be better to do it manually.

Reply:
> Whoops, careful there... How do you know that there won't be a `crypto/ec/curve448/arch_64/f_impl.h`? Will it get the exact same guard? My I suggest something like `OSSL_EC_ARCH_32_F_IMPL_H`?

Naming rules for the header guards have now been refined, see 0556c0d9ae.

The `sed_script`s you see in the patch are the transformation rules starting from the `upper_filename`, which is created from the relative pathname by converting all letters to uppercase and all slashes to underscores.

Sometimes there a `new_guard` is defined explicitely, instead of specifying a sed transformation rule.